src/main/java/com/bt/code/egress/App.java

```diff
@@ -1,9 +1,6 @@
 package com.bt.code.egress;
 
-import com.bt.code.egress.process.FileReplacer;
-import com.bt.code.egress.process.FolderReplacer;
-import com.bt.code.egress.process.LineReplacer;
-import com.bt.code.egress.process.WordReplacer;
+import com.bt.code.egress.process.*;
```

Comment:
Sorry for this change, didn't configure IntelliJ to disable import collapses yet, will fix